### ISSUE TITLE
Backprop gradient sanity is not correct if two conv layers are used

### ISSUE THREAD

**Issue by ddante** [ddante/NilDa#7]
NilDa::layer* l0 = new NilDa::inputLayer({rI, cI, chI});

  NilDa::layer* l1 = new NilDa::conv2DLayer(
                                            nFilters,
                                            {rF, cF},
                                            {rS, cS},
                                            padding,
                                            "relu"
                                           );

  NilDa::layer* l2 = new NilDa::conv2DLayer(
                                            nFilters,
                                            {rF, cF},
                                            {rS, cS},
                                            padding,
                                            "relu"
                                           );

  NilDa::layer* l3 = new NilDa::denseLayer(3, "softmax");
